tests/DCPS/Compiler/is_topic_type/run_test.pl

```diff
@@ -28,6 +28,9 @@
   "NonAnnotatedModule/NestedStruct"                => {not_found => 1, dn => 0, no_dn => 0},
   "NonAnnotatedModule/NestedTrueStruct"            => {not_found => 1, dn => 0, no_dn => 0},
   "NonAnnotatedModule/NestedFalseStruct"           => {not_found => 1, dn => 1, no_dn => 1},
+);
+
+my %expected2 = (
```

Comment:
How about keeping the hashes combined and record the file name to check in `subtest`?